File: Python/40.py
import os
n1=0
n2=0
n3=0
somapara=0
somacu=0
somacili=0
somatudo=0
volpara= 0
volcu=0
volcili=0
i=1
aresta=0
a=0
b=0
c=0
raio=0
altc=0
print("\n\nEsse algoritmo calcula o volume total de armazenamento reservatorio.Com n tanques de formatos diferentes !")
n1=int(input("\n\nDigite o numero e tanques de formato cubico: "))
for i in range(1,n1+1):
    i=i+1
    aresta=float(input("\n\nDigite o valor da aresta : "))
    volcu = (aresta*aresta)*aresta
    somacu = somacu +volcu
n2=int(input("\n\nDigite o numero e tanques de formato de paralelepipedo: "))
for i in range (1,n2+1):
    i=i+1
    a=float(input("\n\nDigite a altura: "))
    b=float(input("\n\nDigite o comprimento: "))
    c=float(input("\n\nDigite a largura: "))
    volpara= (a*b)*c
    somapara = somapara +volpara
n3=int(input("\n\nDigite o numero e tanques de formato de cilindro: "))
for i in range (1,n3+1):
    i=i+1
    alt=float(input("\n\nDigite a altura: "))
    raio=float(input("\n\nDigite o raio: "))
    volcili= (alt*3.14)*raio**2
    somacili = somacili +volcili
somatudo= (somacu + somapara)+somacili
print(f"\n\nEsse é o valor total do reservatorio : {somatudo}")
print("\n\nObrigado por ultilizar  nosso programa")

    
    
    
    
    
    


os.system("pause")

File: C/40.cpp
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main(void){
	setlocale(LC_ALL, "Portuguese");
	int i,n,op,soma,temp;

	printf("Esse programa calcula o volume total de n reservatorios com base em seu formato!\n\n");
	
	printf("\n\nQuantidade de reservatorios: ");
	scanf("%d",&n);
	
	for(i=1;i<=n;i++){
		printf("\n\n Menu\n1-Paralelepipedo\n2-Cubo\n3-Cilindro");
		printf("\n\nFormato do Reservatorio %d: ",i);
		scanf("%d",&op);
		
		switch(op){
			case 1:
				printf("\nVolume do Paralelepipedo: ");
				scanf("%d",&temp);	
				soma=soma+temp;
				break;
			case 2:
				printf("\nVolume do Cubo: ");
				scanf("%d",&temp);	
				soma=soma+temp;
				break;
			case 3:
				printf("\nVolume do Cilindro: ");
				scanf("%d",&temp);	
				soma=soma+temp;
				break;
			default:
				printf("Opção invalida!");
		}
				
	}
	printf("\n\nA O volume total do(s) %d reservatorio(s) é de %d!",n,soma);
	printf("\n\nObrigado por usar nosso sistema!\n\n");
	system("pause");
	return 0;
}


